src/pages/LoginPage.js

```diff
@@ -0,0 +1,120 @@
+
+import '../assets/styles/pages/LoginPage.css';
+
+function LoginPage(){
+    const [emailErrorVisibility, setEmailErrorVisibility] = useState('hidden');
+    const [passwordErrorVisibility, setPasswordErrorVisibility] = useState('hidden');
+
+    const [emailErrorMessage, setEmailErrorMessage] = useState('');
+    const [passwordErrorMessage, setPasswordErrorMessage] = useState('');
+
+    const [currentEmail, setCurrentEmail] = useState('');
+    const [currentPassword, setCurrentPassword] = useState('');
+
+    const handleClick = e => {
+        if(e.target.outerText === 'Sign Up') {
+            /////////////////////////////////
+            //REDIRECT TO SIGN UP PAGE HERE//
+            /////////////////////////////////
+            console.log('REDIRECT TO SIGN UP PAGE')
+            return 0;
+        }
+        e.preventDefault();
+
+        setEmailErrorMessage('');
+        setPasswordErrorMessage('');
+        setEmailErrorVisibility('hidden');
+        setPasswordErrorVisibility('hidden');
+
+        // Validations
+
+        let emailIsValid = true;
+        let passwordIsValid = true;
+
+        let emailHasAnAt = false;
+        let emailHasADomain = true;
+        currentEmail.split('').forEach((char, index, arr) => {
+            if(char === '@'){
+                emailHasAnAt = true;
+            }
+            if(arr.indexOf('@') === arr.length - 1){
+                emailHasADomain = false;
+            }
+        })
+        if(currentEmail === ''){
+            emailIsValid = false;
+            setEmailErrorMessage('Please enter your email');
+            setEmailErrorVisibility('visible');
+        } else if(!emailHasAnAt){
+            emailIsValid = false;
+            setEmailErrorMessage('Invalid email, please add an @ to the email');
+            setEmailErrorVisibility('visible');
+        } else if(!emailHasADomain){
+            emailIsValid = false;
+            setEmailErrorMessage('Invalid email, add a domain next to the @');
+            setEmailErrorVisibility('visible');
+        }
+
+        if(currentPassword.length < 4){
+            passwordIsValid = false;
+            if(currentPassword === ''){
+                setPasswordErrorMessage('Please enter your password');
+                setPasswordErrorVisibility('visible');
+            } else {
+                setPasswordErrorMessage('Invalid password, the password is too short');
+                setPasswordErrorVisibility('visible');
+            }  
+        }
+
+        if(emailIsValid && passwordIsValid){
+            /////////////////////////////
+            //REDIRECT TO HOMEPAGE HERE//
+            /////////////////////////////
+            console.log('REDIRECT TO HOMEPAGE')
+        }
+
+    }
+
+    const handleChange = e => {
+        if(e.target.name === 'login-email'){
+            setCurrentEmail(e.target.value);
+        } else {
+            setCurrentPassword(e.target.value);
+        }
```

Comment:
If you use one hook, you don't need to use conditionals. Use previous and current state to update hooks. For example:

`setState((prevState) => { ...prevState, currentState })`

Reply:
Done, I used this way for state and removed conditionals